dev-libs/marisa/marisa-0.2.6.ebuild

```diff
@@ -1,8 +1,9 @@
-# Copyright 2014-2023 Gentoo Authors
+# Copyright 2014-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI="8"
-PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_COMPAT=( python3_{9..12} )
```

Comment:
While we are at it we can bump off of 3_9 and move to 3_10 as the minimum -- the eclass simply ignores 3_9 if specified anyway.